frontend/src/container/Tracim.jsx

```diff
@@ -89,6 +89,10 @@ class Tracim extends React.Component {
         console.log('%c<Tracim> Custom event', 'color: #28a745', type, data)
         this.props.dispatch(appendBreadcrumbs(data.breadcrumbs))
         break
+      case CUSTOM_EVENT.SET_TITLE:
+        console.log('%c<Tracim> Custom event', 'color: #28a745', type, data)
+        document.title = data.title
```

Comment:
I think we should always show the instance name at the end of the title, and/or always have the word tracim. Please do some research about what should be included in the title.

Reply:
After some research, I think we should too, a lot of web application use `{{pageName}} - {{appName}}` (google drive, OneDrive, gitLab), but what will be the appName ? `Tracim` or the specific instance name ?

Reply:
I don't have the answer, do ask around and investigate to be able to give Damien all the info he needs to decide (pros and cons of both possibilities)